runtime/commonMain/src/kotlinx/serialization/internal/Enums.kt

```diff
@@ -77,4 +76,6 @@ public class EnumSerializer<T : Enum<T>>(
         }
         return values[index]
     }
+
+    override fun toString(): String = "kotlinx.serialization.internal.EnumSerializer<${descriptor.serialName}>"
```

Comment:
We usually do not have `toString()` on serializers, why is it needed?

Reply:
For better debugger experience when debugging formats/serializers

Reply:
AFAIR, we once agreed that we should delegate `KSerializer.toString()` to `descriptor.toString()`, and this function doesn't follow this decision. It also doesn't help much until all serializers will have .toString(), so I suggest removing it for now